Dockerfile

```diff
@@ -13,6 +13,7 @@ COPY . /go/src/github.com/openshift/origin
 ENV GOPATH=/go \
     PATH=$PATH:$GOROOT/bin:$GOPATH/bin
 
+RUN getent hosts mirrorlist.centos.org
```

Comment:
I was thinking we wouldn't merge this but just kick it until it failed -- was merging this and making it part of the images for a while a possiblity?